microk-fo.rkt

```diff
@@ -19,17 +22,20 @@
 (require "common.rkt")
 
 ;; first-order microKanren
-(struct disj    (g1 g2)                  #:prefab)
-(struct conj    (g1 g2)                  #:prefab)
-(struct relate  (thunk description)      #:prefab)
-(struct ==      (t1 t2)                  #:prefab)
-(struct =/=     (t1 t2)                  #:prefab)
-(struct symbolo (t)                      #:prefab)
-(struct stringo (t)                      #:prefab)
-(struct numbero (t)                      #:prefab)
-(struct bind    (bind-s bind-g)          #:prefab)
-(struct mplus   (mplus-s1 mplus-s2)      #:prefab)
-(struct pause   (pause-state pause-goal) #:prefab)
+(struct disj     (g1 g2)                  #:prefab)
+(struct conj     (g1 g2)                  #:prefab)
+(struct relate   (thunk description)      #:prefab)
+(struct ==       (t1 t2)                  #:prefab)
+(struct =/=      (t1 t2)                  #:prefab)
+(struct symbolo  (t)                      #:prefab)
+(struct stringo  (t)                      #:prefab)
+(struct numbero  (t)                      #:prefab)
+(struct !symbolo (t)                      #:prefab)
```

Comment:
Use `not-` rather than `!`

Reply:
https://github.com/DakshChan/first-order-miniKanren/commit/d5efb1eb9e2793c532e937f9bf75be51ffbea36f